user/utils.py

```diff
@@ -0,0 +1,27 @@
+import jwt
+import json
+
+from django.http import JsonResponse
+
+from .models              import User
+from dienstag.my_settings import SECRET, ALGORITHM
+
+def login_decorator(func):
+    def wrapper(self, request, *args, **kwargs):
+        if 'Authorization' not in request.headers:
+            return JsonResponse({'message': 'NEED_LOGIN'}, status=401)
+
+        try:
+            access_token = request.headers['Authorization']
+            payload = jwt.decode(access_token, SECRET, ALGORITHM)
+            login_user = User.objects.get(id=payload['user_id'])
+            request.user = login_user
```

Comment:
```suggestion
            access_token = request.headers['Authorization']
            payload      = jwt.decode(access_token, SECRET, ALGORITHM)
            login_user   = User.objects.get(id=payload['user_id'])
            request.user = login_user
```
위코드 코드 컨벤션 지켜주세요~
다른 부분들도 한 번 더 확인해 주세요

Reply:
위코드 코드 컨벤션 반영했습니다~~!